src/App.js

```diff
@@ -1,26 +1,41 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useEffect } from 'react';
 import useFetch from './useFetch';
+import useDebounce from './useDebounce';
 import './App.css';
 
 function App() {
   let [searchTerm, setSearchTerm] = useState('')
+  const [results, setResults] = useState([])
 
   const handleInput = useCallback((e) => {
     setSearchTerm(e.target.value)
   }, [])
```

Comment:
```suggestion
  }, [setSearchTerm])
```
Also this is a nice read https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
There's some warnings in the terminal conole
![image](https://user-images.githubusercontent.com/1716853/73998274-25b12380-4987-11ea-8cf0-99ff6bc2118c.png)